examples/elvis-operator/elvis_operator.bal

```diff
@@ -5,16 +5,16 @@ function main(string... args) {
 
     string matchExprOutput;
 
-    // Suppose you need to get the string value of x. If x is null, you want 
-    // to assign some known value. This is how you can do it using match 
-    // expression.
+    // If you need to get the string value of x, and if the value of x is null, you may want
```

Comment:
@djkevincr shouldn't it be `nil` and not `null`?

Reply:
I think since this is a value as opposed to type, it s better to use null.